composer.json

```diff
@@ -32,7 +32,8 @@
         "jackalope/jackalope-doctrine-dbal": "1.1.*|1.2.*",
         "doctrine/phpcr-odm": "~1.3",
         "doctrine/phpcr-bundle": "~1.3",
-        "hautelook/alice-bundle": "~0.2|~1.2"
+        "hautelook/alice-bundle": "~0.2|~1.2",
+        "brianium/paratest": "~0.12.0|~0.13.2"
```

Comment:
hmm .. actually what I meant was to have it both in ``suggest`` and ``require-dev`` .. the fact that we have no failures after you removed it from ``require-dev`` indicates that the test doesn't sufficiently cover the feature ..